### PR TITLE
Feature: Littercoin Integration

### PR DESCRIPTION
**Pull request by lley154**
- added remaining transactions (burn, add ada, mint merchant token)
- updated UI including spinner
- renamed node.js scripts for clarity
- imporved error handling routines

Next steps is to do final code cleanup and complete testing

### CODE REVIEW

File: app/Http/Controllers/Littercoin/LittercoinController.php [hunk 1]
```diff
@@ -24,16 +33,255 @@ public function getUsersLittercoin ()
...
+        $response = exec($cmd);
+        $responseJSON = json_decode($response, false);
+
+        if ($responseJSON->status == 200) {
+
+            // Update the amount of littercoin paid to user in the DB
+
+            // Commented out for testing purposes
+            //$user->littercoin_paid = $littercoinPaid + $littercoinDue;
+            //$user->save();
```
Comment (xlcrr) [inline app/Http/Controllers/Littercoin/LittercoinController.php:105]: We should probably update each littercoin in the new table as sent

File: app/Http/Controllers/Littercoin/LittercoinController.php [hunk 2]
```diff
@@ -24,16 +32,270 @@ public function getUsersLittercoin ()
...
+     */
+    public function mintTx (Request $request) {
+
+        $destAddr = $request->input('destAddr');
+        $changeAddr = $request->input('changeAddr');
+        $utxos = $request->input('utxos');
+        $strUtxos=implode(",",$utxos);
+
+        $userId = Auth::user()->id;
+        $littercoinPaid = Auth::user()->littercoin_paid;
```
Comment (xlcrr) [inline app/Http/Controllers/Littercoin/LittercoinController.php:61]: I think we should calculate this from the littercoin table and update each littercoin as sent or not

Comment (lley154) [reply app/Http/Controllers/Littercoin/LittercoinController.php]: agreed, with the latest commits, we are now only reading and update the littercoin table

File: app/Http/Controllers/Littercoin/LittercoinController.php (hunk 1 above)
Comment (lley154) [reply app/Http/Controllers/Littercoin/LittercoinController.php]: this has been completed in recent commits

**Review by xlcrr — APPROVED** [review]

**Pull request merged**